diff --git a/node_modules/@expo/metro-config/build/serializer/serializeChunks.js b/node_modules/@expo/metro-config/build/serializer/serializeChunks.js
index 10c8ce3..5acfef5 100644
--- a/node_modules/@expo/metro-config/build/serializer/serializeChunks.js
+++ b/node_modules/@expo/metro-config/build/serializer/serializeChunks.js
@@ -522,7 +522,6 @@ function gatherChunks(runtimePremodules, chunks, settings, preModules, graph, op
                 const asyncChunks = gatherChunks(runtimePremodules, chunks, { test: pathToRegex(dependency.absolutePath) }, isWorker ? runtimePremodules : [], graph, options, true, isWorker);
                 // Seal all chunks that are for web workers, as these must be self-sufficient chunks
                 if (isWorker) {
-                    (0, assert_1.default)(asyncChunks.size, `Worker chunk not found for: ${dependency.absolutePath}`);
                     for (const chunk of asyncChunks) {
                         chunk.seal();
                     }
